src/main/model/Game.java

```diff
@@ -141,4 +173,48 @@ protected void removeCardFromBoard(Card card) {
     protected boolean isSelectionValid(char selection) {
         return !(selection < 'A' || selection > 'I' || inPlay[selection - ASCII_OFFSET] == null);
     }
+
+    public void endMessages(){
+        if(isWon()){
+            System.out.println("Congratualations you win! :) \n");
+            movesPlayback();
+
+        }
+        if(isStalemate()) {
+            System.out.println("Unfortunately this time you have lost! :( \n");
+            movesPlayback();
+        }
+    }
+
+    public void movesPlayback(){
+        System.out.println("would you like to see your moves back? (y/n) : ");
```

Comment:
`Would you like to see a replay?` is probably better English imo

Reply:
> `Would you like to see a replay?` is probably better English imo

that can be something we can add in at the end sure. it wont brake anything tbf but you are right